### PR TITLE
Fix #1553 Made wave 1/2 labels clickable in Wave Generator

### LINKED ISSUE #1553: Wave 1/Wave 2 labels should be clickable in wave generator
**Actual Behaviour**

Currently we can toggle between wave 1 and wave 2 only using the buttons provided.

**Expected Behaviour**

Labels of wave 1/ wave 2 should be clickable to toggle between 2 waves

**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by neel1998**
Fixes #1553 

**Changes**:

Wave Labels in wave generator are now clickable

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**: 
[wave_labels.zip](https://github.com/fossasia/pslab-android/files/2881082/wave_labels.zip)

### CODE REVIEW

File: app/src/main/java/io/pslab/activity/WaveGeneratorActivity.java [hunk 1]
```diff
@@ -228,7 +231,16 @@ public void onClick(View view) {
                 }
             }
         });
-
+        //wave1 label clickable
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/activity/WaveGeneratorActivity.java]: Remove this comment

File: app/src/main/java/io/pslab/activity/WaveGeneratorActivity.java [hunk 2]
```diff
@@ -238,7 +250,16 @@ public void onClick(View view) {
                 }
             }
         });
-
+        //wave2 label clickable
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/activity/WaveGeneratorActivity.java]: And this one too

File: app/src/main/java/io/pslab/activity/WaveGeneratorActivity.java [hunk 3]
```diff
@@ -184,6 +184,9 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_wave_generator_main);
         ButterKnife.bind(this);
 
+
+        waveMonWave1.setEnabled(true);
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/activity/WaveGeneratorActivity.java]: Do we need to enable them here as the property is removed from xml file?

**Comment by neel1998** [comment]
@CloudyPadmal  kindly review , i have made the changes

**Comment by harsh-2711** [comment]
@neel1998 Please solve the merge conflicts

**Comment by neel1998** [comment]
@CloudyPadmal  kindly review

**Comment by neel1998** [comment]
@CloudyPadmal  kindly review and merge.

**Comment by cweitat** [comment]
@neel1998 please stop tagging Padmal everyday. This is a community project. If you are urgent to for someone to approve your changes, please ask in the public chat channel.

**Comment by neel1998** [comment]
@cweitat  will keep this in mind from next time. Sorry for the inconvenience

**Comment by harsh-2711** [comment]
@neel1998 Can you please post a screenshot or GIF of the changes made? From next time please try to keep it in your PR message.

**Comment by mariobehling** [comment]
Please follow best practices and provide screenshots. Merging this as WIP.

**Pull request merged**